apps/openssl.cnf

```diff
@@ -60,6 +60,11 @@ default = default_sect
 # included fipsmodule.cnf.
 # fips = fips_sect
 
+# XXX If no providers are activated, the default one is activated implicitly.
+# If you add a section explicitly activating any other providers,
+# you MAY need to explicitly activate the default provider, otherwise
```

Comment:
`you most probably need...` might be more suggestive?

Reply:
Fixed, thanks!